README.md

```diff
@@ -60,7 +59,7 @@ See [Project Logistics](projects/logistics.md) for important dates and deliverab
 * [IT Helpouts](projects/it-helpouts.md) - **Team:** [Johan Adami](people/johan-adami.md), [Maria Shen](people/maria-shen.md), [Hannah Xue](people/hannah-xue.md)
 * [Job Tracker](projects/job-tracker.md) - **Team:** [Srinivasa Teja](people/srinivasa-teja.md), [Bhagyasri Canumalla](people/bhagyasri-canumalla.md), [Sindhu Banavara Ravindra](people/sindhu-banavara-ravindra.md)
 * [Meeting Point](projects/meeting-point.md) - **Team:** [Inna Shteinbuk](people/inna-shteinbuk.md), [Jocelyn Kong](people/jocelyn-kong.md), [Halil Akin](people/halil-akin.md)
-* [MeetMe](projects/meetme.md) - **Team** [Mark Assad](people/mark-assad.md), [Anas Bouzoubaa](people/anas-bouzoubaa.md), [Zaid Haque](people/zaid-haque.md)
+* [RendezVous](projects/rendezvous.md) - **Team** [Mark Assad](people/mark-assad.md), [Anas Bouzoubaa](people/anas-bouzoubaa.md), [Zaid Haque](people/zaid-haque.md)
```

Comment:
no only updated our project name and added a few links including our code

On Fri, Dec 11, 2015 at 8:40 AM, Daniel Doubrovkine (dB.) @dblockdotorg <
notifications@github.com> wrote:

> In README.md
> <https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/303#discussion_r47355288>
> :
>
> > @@ -60,7 +59,7 @@ See [Project Logistics](projects/logistics.md) for important dates and deliverab
> >  * [IT Helpouts](projects/it-helpouts.md) - **Team:** [Johan Adami](people/johan-adami.md), [Maria Shen](people/maria-shen.md), [Hannah Xue](people/hannah-xue.md)
> >  * [Job Tracker](projects/job-tracker.md) - **Team:** [Srinivasa Teja](people/srinivasa-teja.md), [Bhagyasri Canumalla](people/bhagyasri-canumalla.md), [Sindhu Banavara Ravindra](people/sindhu-banavara-ravindra.md)
> >  * [Meeting Point](projects/meeting-point.md) - **Team:** [Inna Shteinbuk](people/inna-shteinbuk.md), [Jocelyn Kong](people/jocelyn-kong.md), [Halil Akin](people/halil-akin.md)
> > -* [MeetMe](projects/meetme.md) - **Team** [Mark Assad](people/mark-assad.md), [Anas Bouzoubaa](people/anas-bouzoubaa.md), [Zaid Haque](people/zaid-haque.md)
> > +* [RendezVous](projects/rendezvous.md) - **Team** [Mark Assad](people/mark-assad.md), [Anas Bouzoubaa](people/anas-bouzoubaa.md), [Zaid Haque](people/zaid-haque.md)
>
> Is this undoing someone else's change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/303/files#r47355288>
> .
>